content/webapp/pages/article.tsx

```diff
@@ -228,10 +228,22 @@ const ArticlePage: FC<Props> = ({ article, jsonLd }) => {
                   </Space>
                 </Fragment>
               ))}
-            {readingTime && article.readingTime ? (
+            {!readingTime && article.readingTime ? (
               <>
                 <span className={font('intr', 6)}>
-                  <span className={font('intr', 5)}>| </span> reading time{' '}
+                  <span className={font('intr', 5)}>
+                    {' '}
+                    <Space
```

Comment:
[Alex shared a nice idea for those](https://github.com/wellcomecollection/wellcomecollection.org/pull/8519#discussion_r977915695), maybe it's of interest? 
([the commit with my changes](https://github.com/wellcomecollection/wellcomecollection.org/pull/8519/commits/25d04636351e79673579541f44447a2d60ad21b5))

Reply:
I know yours aren't conditional so you don't need all of it, but it also looks cleaner in the codebase and it doesn't need to be detected by a reader, so an idea!